pp_calibrate.py

```diff
@@ -478,14 +478,26 @@ def calibrate(filenames, minstars, manfilter, manualcatalog,
     # read in ldac data into catalogs
     catalogs, filternames = [], {}
     for filename in filenames:
-        hdulist = fits.open(filename, ignore_missing_end=True)
+        hdulist = fits.open(filename, mode='update', ignore_missing_end=True)
+        
+        # add flag keyword to header and set it to STARTED
+        header = hdulist[0].header
+        try:
+            header.set('PP_CALIB', 'STARTED', 'PP: pp_calibrate status flag',
+                        after='PP_PHOTO')
+        except:
```

Comment:
If possible, please try to avoid `except` without listing a specific error that you would like to intercept. What kind of error could occur that would need to be intercepted here? I guess the only issue might be that `PP_PHOTO` does not exist, so that would probably be a `KeyError`. Anything else?